app/view_objects/results_view.rb

```diff
@@ -202,6 +203,18 @@ def subjects
 
 private
 
+  def ordered_sites_for(course)
```

Comment:
This is called `ordered_sites_for`, but it doesn't sort/order the sites it extracts. How about calling it `new_or_running_sites_for`?

`#namingishard`

Reply:
👍